db/migrate/20230703183221_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.string :user
```

Comment:
This should be lined to user table

Reply:
If I need to reply to a comment, is this possible right now? if now how will you go about it?